platform/vs/docker-sonic-vs.mk

```diff
@@ -11,7 +11,8 @@ $(DOCKER_SONIC_VS)_DEPENDS += $(SYNCD_VS) \
                               $(LIBYANG_CPP) \
                               $(LIBYANG_PY3) \
                               $(SONIC_UTILITIES_DATA) \
-                              $(SONIC_HOST_SERVICES_DATA)
+                              $(SONIC_HOST_SERVICES_DATA) \
+                              $(FRAMEWORK)
```

Comment:
@rkavitha-hcl What is the use case for having the framework service in `docker-sonic-vs`?